server/sonar-web/src/main/js/apps/quality-gates/components/Projects.js

```diff
@@ -18,44 +18,55 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import React from 'react';
-import ProjectsView from '../views/gate-projects-view';
+import escapeHtml from 'escape-html';
+import SelectList from '../../../components/SelectList';
+import { translate } from '../../../helpers/l10n';
+import { getBaseUrl } from '../../../helpers/urls';
 
 export default class Projects extends React.PureComponent {
   componentDidMount() {
-    this.renderView();
+    this.renderSelectList();
   }
 
-  componentWillUpdate() {
-    this.destroyView();
-  }
-
-  componentDidUpdate() {
-    this.renderView();
-  }
+  renderSelectList = () => {
+    if (!this.container) return;
 
-  componentWillUnmount() {
-    this.destroyView();
-  }
+    const { qualityGate, edit, organization } = this.props;
 
-  destroyView() {
-    if (this.projectsView) {
-      this.projectsView.destroy();
+    const extra = { gateId: qualityGate.id };
+    let orgQuery = '';
+    if (organization) {
+      extra.organization = organization;
+      orgQuery = '&organization=' + organization;
     }
-  }
-
-  renderView() {
-    const { qualityGate, edit, organization } = this.props;
 
-    this.projectsView = new ProjectsView({
-      qualityGate,
-      edit,
-      container: this.refs.container,
-      organization
+    // eslint-disable-next-line no-new
+    new SelectList({
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either remove this useless object instantiation of "SelectList" or use it ([javascript:ConstructorFunctionsForSideEffects](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AConstructorFunctionsForSideEffects))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fless-backbone-more-react&issues=AWG-K8_X_Zsgh39atBHn&open=AWG-K8_X_Zsgh39atBHn)

[ISSUE_KEY]: AWG-K8_X_Zsgh39atBHn